src/main.py

```diff
@@ -27,6 +28,38 @@ def get_json_from_file(file_path: str) -> dict:
 
     raise FileExistsError(f"Could not locate resource: {file_path}")
 
+def run_strategy(json_settings):
+    """
+    Function to execute the stategy in main
+    :param json_settings: json of project settings
+    :param credentials: json of user credentials
+    :return: Boolean. True if strategy ran successfully with no errors. Else False.
+    """
+    # Get symbols array from settings.json
+    symbols_arr = json_settings["mt5"]["symbols"]
+
+    # Get timeframe from settings.json
+    timeframe=json_settings["mt5"]["timeframe"]
+
+    # Initialize all symbols
+    for symbol in symbols_arr:
+        try:
+            trader.initialize_symbol(symbol)
+        except Exception as e:
+            print(e)
+    
+    # Get a table of ema calculations for every initialized symbol
+    for symbol in symbols_arr:
+
+        ema_x_strategy_table = strats.ema_cross_strategy(symbol, timeframe, 50, 200, 10000, 0.03)
+
+        # Console output
+        if (ema_x_strategy_table):
+            print(f"Trade made on {symbol}.")
```

Comment:
Is it possible to list what kind of trade (buy or sell)?

Reply:
I'll look into this comment

Reply:
The only function which knows the type of order in our code is the "place_order" function, we can have it print something or return a value other than boolean upstream. I can see why knowing the type of order would be valuable. Can we discuss next time we meet?